pkg/features/embedded/elemental-rootfs/usr/lib/dracut/modules.d/30elemental-rootfs/module-setup.sh

```diff
@@ -24,15 +24,10 @@ install() {
     declare systemdsystemunitdir=${systemdsystemunitdir}
 
     inst_multiple \
-        mount mountpoint sort rmdir findmnt rsync cut realpath basename lsblk
+        mount cut basename lsblk
 
     inst_simple "/etc/elemental/config.yaml"
 
-    # Include utilities required for elemental-setup services,
-    # probably a devoted dracut module makes sense
-    inst_multiple -o \
```

Comment:
There was actually a an elemental-setup dracut module, this comment is was so outdated and all this dependencies should have been moved to the devoted dracut module.